docs/Governance.md

```diff
@@ -119,30 +121,46 @@ Responsibilities:
 ## Project Milestones
 
 eBPF for Windows uses monthly milestones in the format `YYMM`. So, e.g., the milestone for June, 2024 will be called 2406.
-At any time, there are four active milestones in the project. There will be a milestone for the current calendar month (`N`),
-and two others for the next two months (`N+1`), (`N+2`). And there is always a `Backlog` milestone for backlog issues to be
-addressed in the future. In the first triage meeting of a month, the previous milestone must be closed and a new `N+2` milestone must be created.
-
-During the weekly triage meetings, whenever any issue is marked with the "triaged" label, triage drivers MUST set an issue to one of the four milestones
-according to the priority of the issues.
+At any time, there will be one current milestone (for the current month) and one or more future milestones, in the project.
+A `Backlog` milestone exists for backlog issues.
+In the weekly triage meetings, new issues should be added to a milestone following guidelines in the
+[Prirotizing Issues](Governance.md#prioritizing-issues) section.
 
 ## Prioritizing Issues
 
 To help prioritize issues, the triage drivers and maintainers MUST use *special labels* to prioritize issues as follows:
 ### P1:
 This label represents highest priority. This label should be used for bugs impacting users, new features, or enhancements that are urgently
-requested by users. `P1` issues MUST be set to the *current milestone (`N`)*. If there is not enough time left in the current milestone to address
-the `P1` issue, it must be set to the *next milestone (`N+1`)*.
+requested by users. `P1` issues MUST be set to the *current milestone*.
```

Comment:
I don't understand why the additional sentence was removed.  Let's discuss in a meeting.

Reply:
This is addressed in the following sections.